psi4/src/psi4/dlpno/mp2.cc

```diff
@@ -791,8 +800,8 @@ void DLPNOMP2::pno_transform() {
             int centerq = ribasis_->function_to_center(q);
             for (int a_ij = 0; a_ij < npao_ij; a_ij++) {
                 int a = lmopair_to_paos_[ij][a_ij];
-                i_qa->set(q_ij, a_ij, qia_[q]->get(riatom_to_lmos_ext_dense_[centerq][i], a));
-                j_qa->set(q_ij, a_ij, qia_[q]->get(riatom_to_lmos_ext_dense_[centerq][j], a));
+                i_qa->set(q_ij, a_ij, qia_[q]->get(riatom_to_lmos_ext_dense_[centerq][i], riatom_to_paos_ext_dense_[centerq][a]));
```

Comment:
I don't understand this line. Because this is a dense map, it could in principle have a value of `-1`. What happens if `riatom_to_lmos_ext_dense_[centerq][i] = -1`?